cfgov/unprocessed/css/video-player.less

```diff
@@ -57,6 +57,15 @@
     text-align: center;
 }
 
+.video-player_image {
+    opacity: 1;
+    transition: opacity 2s;
```

Comment:
Yeah, you might want to add the capability to pass an options object which allowed you to specify duration and time.